huaweicloud/services/acceptance/bcs/resource_huaweicloud_bcs_instance_test.go

```diff
@@ -171,6 +171,36 @@ func testAccCheckBCSInstanceExists(name string, instance *blockchains.BCSInstanc
 	}
 }
 
+func TestAccBCSInstance_updateWithEpsId(t *testing.T) {
+	var instance blockchains.BCSInstance
+	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	password := fmt.Sprintf("%s%s%d", acctest.RandString(5), acctest.RandStringFromCharSet(3, "!@$%^-_=+[{}]:,./?"),
+		acctest.RandIntRange(1, 3))
+	resourceName := "huaweicloud_bcs_instance.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheckEpsID(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckBCSInstanceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testBCSInstance_basic(rName, password),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckBCSInstanceExists(resourceName, &instance),
+					resource.TestCheckResourceAttr(resourceName, "enterprise_project_id", acceptance.HW_ENTERPRISE_PROJECT_ID_TEST),
+				),
+			},
+			{
+				Config: testBCSInstance_basicTestWithEps(rName, password),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckBCSInstanceExists(resourceName, &instance),
+					resource.TestCheckResourceAttr(resourceName, "enterprise_project_id", "0"),
```

Comment:
suggest to use `HW_ENTERPRISE_MIGRATE_PROJECT_ID_TEST` instead of `0`

Reply:
done